Assets/MRTK/Tools/MSBuild/Scripts/AssetScriptReferenceRetargeter.cs

```diff
@@ -239,7 +239,12 @@ private static async Task ProcessYamlFile(string filePath, string targetPath, Di
                             }
                             else if (nonClassDictionary.ContainsKey(guid))
                             {
-                                throw new InvalidDataException($"A script without a class ({nonClassDictionary[guid]}) is being processed.");
+                                // this guid bypasses throwing the exception. The reason for this is that there is currently an asset (MRTK-OculusConfig.asset) that is reliant
+                                // on Unity 2019+ specific code (MRTKOculusConfig.cs), which causes CI to fail since it's running on Unity 2018.
+                                if (guid != "17acf00f59af96c4bb501d1f9620d1c4")
```

Comment:
If you wanna get really fancy, you could actually wrap this up in a #if Unity2018 version thingy.

I'm fine either way since we have this great comment here.

Reply:
I mean I assume this would work but obviously I haven't tested myself!